atproto/xrpc_client/models/app/bsky/actor/get_suggestions.py

```diff
@@ -22,6 +23,11 @@ class Params(base.ParamsModelBase):
     limit: t.Optional[int] = Field(default=50, ge=1, le=100)  #: Limit.
 
 
+class ParamsDict(te.TypedDict):
+    cursor: te.NotRequired[t.Optional[str]]  #: Cursor.
```

Comment:
can't it be just NotRequired[str]?

Reply:
It could, but given that the corresponding model could be created by either not supplying a value, or by passing None, I'm inclined to keep the typeddict the same way.